tests/settings.py

```diff
@@ -1,32 +1,52 @@
 import os
+import sys
 
 import dj_database_url
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+if sys.argv[0] != "test":
```

Comment:
Nit: This check should probably be at the bottom, so it supersedes the other settings.

Reply:
Done